evalml/automl/utils.py

```diff
@@ -68,3 +69,19 @@ def make_data_splitter(X, y, problem_type, problem_configuration=None, n_splits=
             raise ValueError("problem_configuration is required for time series problem types")
         return TimeSeriesSplit(n_splits=n_splits, gap=problem_configuration.get('gap'),
                                max_delay=problem_configuration.get('max_delay'))
+
+
+def tune_binary_threshold(pipeline, objective, problem_type, X_threshold_tuning, y_threshold_tuning):
```

Comment:
Just curious, is `y_threshold_tuning` a preoptimization? We currently don't have anything that does this right?

Reply:
Hm, not sure I follow. We need both the features and the target labels in order to compute the "optimal" threshold value.

Reply:
Oops, I didn't see the line using y_threshold_tuning and was confused about whether or not it was used. My bad!